pkg/cmd/version/exec.go

```diff
@@ -0,0 +1,79 @@
+// Copyright Contributors to the Open Cluster Management project
+package version
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	apps "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/spf13/cobra"
+	clusteradm "open-cluster-management.io/clusteradm"
+	"open-cluster-management.io/clusteradm/pkg/helpers"
+)
+
+func (o *Options) complete(cmd *cobra.Command, args []string) (err error) {
+	return nil
+}
+
+func (o *Options) validate() error {
+	return nil
+}
+func (o *Options) run() (err error) {
+	fmt.Printf("client\t\tversion\t:%s\n", clusteradm.GetVersion())
+	client, err := helpers.GetControllerRuntimeClientFromFlags(o.ConfigFlags)
+	if err != nil {
+		return err
+	}
+	return o.runWithClient(client)
+}
+
+func (o *Options) runWithClient(client crclient.Client) (err error) {
+	cms := &corev1.ConfigMapList{}
```

Comment:
let's empty  this function. I think it tries to get image-manfest in ACM which will not work.

Reply:
it works, if image-manifest not present the list is empty and so jump here: https://github.com/itdove/clusteradm/blob/bd9c173b4231918e5a3928d85e590c584cb8ce6f/pkg/cmd/version/exec.go#L68

on kind:

(base) dvernier@dvernier-mac ~ % clusteradm version
client		version	:0.0.1
server release	version	:v1.19.7

As per the arch call this morning, I understood that we have to do function that works for both products.

Reply:
There is also a e2e test on it which run on kind: https://github.com/itdove/clusteradm/blob/6ca5416204fd1b207082bc103695a03232adbe32/build/run-functional-tests.sh#L20

Reply:
i think we should remove wording like `acm-custom-registry` from the code. 
If the intention is to fetch the server version. We need  a  better approach instead  of reading it from image manifest...So it is better to make this an empty function at first.

Reply:
I will just remove the ACM part of it